python/lib/dependabot/python/file_fetcher.rb

```diff
@@ -139,7 +139,7 @@ def python_version_file
 
         # Check the top-level for a .python-version file, too
         reverse_path = Pathname.new(directory[0]).relative_path_from(directory)
-        @python_version_file ||=
```

Comment:
any reason to remove the `||`?

Reply:
There's a check above for `nil`, and `||=` is just shorthand for `a = a || expensive_operation` so sorbet considers the left hand side dead code.

Reply:
||